### ISSUE TITLE
Remove old legal document

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4188]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4188|
|Type|task|
|Created|2009-12-25 15:42:37Z|
|Branch|https://github.com/twisted/twisted/tree/contrib-agreement-4188|

[doc/core/development/legal/contributor-agreement.xhtml](doc/core/development/legal/contributor-agreement.xhtml) seems to be an old document which doesn't apply anymore with Twisted's MIT license. Let's remove it since new contributors don't need to sign this document anymore (at least I didn't and wasn't made aware of this document either).

<details><summary>Searchable metadata</summary>

```
trac-id__4188 4188
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_contrib_agreement_4188 branches/contrib-agreement-4188
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1261755757000000 1261755757000000
changetime__1264016123000000 1264016123000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [28014]) Branching to 'contrib-agreement-4188'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [28015]) Remove references to development/legal, refs [#4188](https://github.com/twisted/twisted/issues/4188)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_5"></a>|@khorn set owner to @thijstriemstra|
|-|-|

Looks good, merge it!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [28035]) Merge contrib-agreement-4188: Remove the old contributor agreement document that isn't
relevant anymore because Twisted uses the MIT license now.

Author: thijs
Reviewer: khorn
Fixes: [#4188](https://github.com/twisted/twisted/issues/4188)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
